xbmc/dialogs/GUIDialogMediaFilter.cpp

```diff
@@ -44,88 +44,164 @@
-//  { "albums",       FieldArtist,        557,    SettingType::List,    "list",   "string",   CDatabaseQueryRule::OPERATOR_EQUALS },
-  { "albums",       FieldAlbumArtist,   566,    SettingType::List,    "list",   "string",   CDatabaseQueryRule::OPERATOR_EQUALS },
-  { "albums",       FieldSource,      39030,    SettingType::List,    "list",   "string",   CDatabaseQueryRule::OPERATOR_EQUALS },
-  { "albums",       FieldRating,        563,    SettingType::Number,  "range",  "number",   CDatabaseQueryRule::OPERATOR_BETWEEN },
-  { "albums",       FieldUserRating,    38018,  SettingType::Integer, "range",  "integer",  CDatabaseQueryRule::OPERATOR_BETWEEN },
-  { "albums",       FieldAlbumType,     564,    SettingType::List,    "list",   "string",   CDatabaseQueryRule::OPERATOR_EQUALS },
-  { "albums",       FieldYear,          562,    SettingType::Integer, "range",  "integer",  CDatabaseQueryRule::OPERATOR_BETWEEN },
-  { "albums",       FieldGenre,         515,    SettingType::List,    "list",   "string",   CDatabaseQueryRule::OPERATOR_EQUALS },
-  { "albums",       FieldMusicLabel,    21899,  SettingType::List,    "list",   "string",   CDatabaseQueryRule::OPERATOR_EQUALS },
-  { "albums",       FieldCompilation,   204,    SettingType::Boolean, "toggle", "",         CDatabaseQueryRule::OPERATOR_FALSE },
-
-  { "songs",        FieldTitle,         556,    SettingType::String,  "edit",   "string",   CDatabaseQueryRule::OPERATOR_CONTAINS },
-  { "songs",        FieldAlbum,         558,    SettingType::List,    "list",   "string",   CDatabaseQueryRule::OPERATOR_EQUALS },
-  { "songs",        FieldArtist,        557,    SettingType::List,    "list",   "string",   CDatabaseQueryRule::OPERATOR_EQUALS },
-  { "songs",        FieldTime,          180,    SettingType::Integer, "range",  "time",     CDatabaseQueryRule::OPERATOR_BETWEEN },
-  { "songs",        FieldRating,        563,    SettingType::Number,  "range",  "number",   CDatabaseQueryRule::OPERATOR_BETWEEN },
-  { "songs",        FieldUserRating,    38018,  SettingType::Integer, "range",  "integer",  CDatabaseQueryRule::OPERATOR_BETWEEN },
-  { "songs",        FieldYear,          562,    SettingType::Integer, "range",  "integer",  CDatabaseQueryRule::OPERATOR_BETWEEN },
-  { "songs",        FieldGenre,         515,    SettingType::List,    "list",   "string",   CDatabaseQueryRule::OPERATOR_EQUALS },
-  { "songs",        FieldPlaycount,     567,    SettingType::Integer, "range",  "integer",  CDatabaseQueryRule::OPERATOR_BETWEEN },
-  { "songs",        FieldSource,      39030,    SettingType::List,    "list",   "string",   CDatabaseQueryRule::OPERATOR_EQUALS }
-};
+    {"movies", FieldTitle, 556, SettingType::String, "edit", "string",
+     CDatabaseQueryRule::OPERATOR_CONTAINS},
+    {"movies", FieldRating, 563, SettingType::Number, "range", "number",
+     CDatabaseQueryRule::OPERATOR_BETWEEN},
+    {"movies", FieldUserRating, 38018, SettingType::Integer, "range", "integer",
```

Comment:
I'm sure that we do not want to let Clang reformat the code like this!
@the-black-eagle this is a useful "error", but can you undo it while the team discusses how to stop Clang/change the code guidlines

Reply:
Yeah, I was annoyed/gutted/words I can't post ! when I saw what it did !  For ref, the command needs to be `git rebase -x 'git clang-format-6.0 HEAD~' upstream/master`.  

Anyway, I'll see what I can do to reverse it. In the meantime, please accept my apologies.  I can do it by hand if/when I've rolled it back but it'll take a few hours.  My IDE can apply the clang-formatting no problem, it's just finding the right bits of code again to apply it to.

Reply:
The command works for me as is, but on my system clang-format-9 is used. Out of curiousity, before your reset everything, could you post the patchset it produced.

If it formated stuff not in your changes, something went wrong with the format.

Reply:
Nevermind - I have seen it. Yes from current pov of code formating this is correct even if it looks odd. Sorry for the trouble - you hit an area where old coding style glitches very heavily with the new design rules.

Reply:
If you are on a Debian based system, you have to install `clang-format` package, which depends on the default clang-format version, e.g. `clang-format-6.0` or `clang-format-9` and creates a symlink to it. That's the same as for clang, clang-tidy, gcc or g++ and many other packages.